core/db/entities.py

```diff
@@ -1,19 +1,40 @@
 import uuid  # noqa
 
-from sqlalchemy import Column, String
+import sqlalchemy
 
 from core.db import db
 from core.db.types import GUID
 
 
+class Organisation(db.Model):
+    __tablename__ = "organisation_dim"
+
+    organisation_id = sqlalchemy.Column(GUID(), default=uuid.uuid4, primary_key=True)
+    organisation_name = sqlalchemy.Column(
```

Comment:
Can we just call these e.g. `id` and `name` as context is derivable from the table name.

Reply:
In my experience that can become confusing quickly when dealing with multiple tables with the same field name and deeply nested or complex queries. Verbose is good sometimes. 

Sometimes the context of the table name can be lost, take the example of simple test:
```
 read_org = Organisation.query.first()
 assert read_org.organisation_name == "Evil Corp"
```
This is obvious as they are right next to each other, but that var on the first line could be named anything, and that's all it takes to loose context. whereas organisation_name always means the same thing.

Reply:
I think there is value in e.g. using `id` consistently for the primary key of tables. So if we can accept that I may compromise on the other fields.
I would hope in real usage we would be returning e.g. Data classes rather than individual plucked columns so the context should also be described from that.